Broken import in v6

## Expected Behavior

zx to run with existing scripts without errors.

## Actual Behavior

When upgrading to v6.0.0, our existing script errored with the following stack trace:

```
node:internal/errors:464
    ErrorCaptureStackTrace(err);
    ^

Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/Users/willo/.nvm/versions/node/v17.4.0/lib/node_modules/zx/src/globals.mjs' imported from /Users/willo/.nvm/versions/node/v17.4.0/lib/node_modules/zx/zx.mjs
    at new NodeError (node:internal/errors:371:5)
    at finalizeResolution (node:internal/modules/esm/resolve:394:11)
    at moduleResolve (node:internal/modules/esm/resolve:944:10)
    at defaultResolve (node:internal/modules/esm/resolve:1041:11)
    at ESMLoader.resolve (node:internal/modules/esm/loader:530:30)
    at ESMLoader.getModuleJob (node:internal/modules/esm/loader:251:18)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:79:40)
    at link (node:internal/modules/esm/module_job:78:36) {
  code: 'ERR_MODULE_NOT_FOUND'
}
```

Version v5 and below appears to function normally. I'll try and create a minimal test case tomorrow morning.

## Steps to Reproduce the Problem

1.
1.
1.

## Specifications

- Version: v6.0.0
- Platform: Mac OS and Ubuntu

--- a/package.json
+++ b/package.json
@@ -45,6 +45,7 @@
     "registry": "https://wombat-dressing-room.appspot.com"
   },
   "files": [
+    "src",
     "*.mjs",
     "*.d.ts"
   ],

